lib/jellyfish_web/api_spec/component/file.ex

```diff
@@ -39,6 +45,12 @@ defmodule JellyfishWeb.ApiSpec.Component.File do
           type: :string,
           description: "Path to track file. Must be either OPUS encapsulated in Ogg or raw h264",
           example: "/root/video.h264"
+        },
+        framerate: %Schema{
+          type: :integer,
+          description: "Framerate of video in a file. It is only valid for video track",
+          nullable: true,
+          example: 30
```

Comment:
Actually, since we publicly declare the default is `30` (in the docs), we can do it here as well.
```suggestion
          default: 30
```

Reply:
I am not sure about that as for FileComponent with audio the default will be `nil`.

Reply:
Yeah, right 👍🏽